combat-trainer.lic

```diff
@@ -255,10 +260,10 @@ class LootProcess
       check_skinning(DRRoom.dead_npcs.first, game_state)
     end
     while 'and get ready to search it' == bput('loot', 'You search', 'I could not find what you were referring to', 'and get ready to search it')
+      @loot_timer = Time.now
```

Comment:
I think this needs to be moved outside of the while loop. If you loot and get 'You search', the while exits without running the code inside, so `@loot_timer` never gets updated.